app/routes/$locale/blog/tag/$tag.tsx

```diff
@@ -58,14 +66,7 @@ export default function Index() {
 
 export function CatchBoundary() {
   const caught = useCatch();
-  return (
-    <div className="container mx-auto mt-32">
-      <h2>Oh noez! We failed.</h2>
-      <p>
-        {caught.status}: {caught.statusText}
-      </p>
-    </div>
-  );
+  return <BasicCatchBoundary {...caught} />;
```

Comment:
Love the new emoji-errors :D Could we use `<BasicCatchBoundary/>` in the other places where we also display such a message?

<img width="405" alt="Screenshot 2022-09-02 at 09 15 26" src="https://user-images.githubusercontent.com/88032788/188080217-f064b18c-6587-4ff1-889c-c1955a55ab3f.png">